repos/backend/endpoints/files.js

```diff
@@ -66,16 +66,17 @@ const parentNodeExists = (nodes, parentPath, newItem) => {
 
 /**
  * Gets the metadata of a path from the local filesystem
- * @param {string} path - full path to the folder or file i.e '/keg/tap/tests/bdd/features'
+ * @param {string} filePath - full path to the folder or file i.e '/keg/tap/tests/bdd/features'
  * 
  * @returns {Object} - Meta data containing {name, parent, type ( folder || file )} properties
  */
-const getPathMeta = path => {
-  const pathSplit = path.split('/')
+const getPathMeta = filePath => {
```

Comment:
Lol... see my above comment. I'm guessing you had to change this because of the `path` require on line 9

Reply:
lol yup